app/models/PeriodBreakdown.scala

```diff
@@ -38,30 +39,51 @@ sealed trait FurloughBreakdown extends PeriodBreakdown {
 sealed trait NicBreakdown extends PeriodBreakdown {
   val topUpPay: Amount
   val additionalPay: Amount
+  val threshold: Threshold
+  val nicCap: NicCap
 }
 
-sealed trait PensionBreakdown extends PeriodBreakdown
+sealed trait PensionBreakdown extends PeriodBreakdown {
+  val threshold: Threshold
```

Comment:
These should be `def` not `val`.  Details [here](https://stackoverflow.com/questions/19642053/when-to-use-val-or-def-in-scala-traits).

Reply:
addressed in latest commit